engine/src/flutter/lib/web_ui/lib/ui_web/src/ui_web/testing.dart

```diff
@@ -35,12 +35,7 @@ final class TestEnvironment {
       keepSemanticsDisabledOnUpdate = true,
       defaultToTestUrlStrategy = true;
 
-  const TestEnvironment.production()
-    : ignorePlatformMessages = false,
-      forceTestFonts = false,
-      disableFontFallbacks = false,
-      keepSemanticsDisabledOnUpdate = false,
-      defaultToTestUrlStrategy = false;
+  const TestEnvironment.production() : this();
```

Comment:
Ugh. I guess the only issue with this is you now need to be careful changing the defaults or adding this to the default ctor. Where before it was separate and clear. 🤷